src/atomate2/vasp/flows/lobster.py

```diff
@@ -154,18 +139,91 @@ def make(
             optimization_uuid=optimization_uuid,
             static_dir=lobster_static_dir,
             static_uuid=lobster_static_uuid,
-            preconverge_static_dir=preconverge_static_dir,
-            preconverge_static_uuid=preconverge_static_uuid,
         )
         jobs.append(lobster_jobs)
 
         # delete all WAVECARs that have been copied
+        # TODO:  this has to be adapted as well
         if self.delete_wavecars:
             delete_wavecars = delete_lobster_wavecar(
                 dirs=lobster_jobs.output["lobster_dirs"],
                 lobster_static_dir=lobster_static.output.dir_name,
-                preconverge_static_dir=preconverge_static_dir,
             )
             jobs.append(delete_wavecars)
 
         return Flow(jobs, output=lobster_jobs.output)
+
+
+@dataclass
+class LobsterUniformMaker(Maker):
+    """
+    Maker to generate uniform VASP band structure run specifically adapted to Lobster.
+
+    This is a static calculation with settings for Lobster runs
+    followed by a uniform non-self-consistent field
+    calculation that generates a WAVECAR.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    static_maker : .BaseVaspMaker
+        The maker to use for the static calculation with special lobster settings.
+    bs_maker : .BaseVaspMaker
+        The maker to use for the non-self-consistent field calculations
+         with special lobster settings.
+    """
+
+    name: str = "uniform lobster structure"
+    static_maker: BaseVaspMaker = field(
+        default_factory=lambda: StaticMaker(
+            input_set_generator=StaticSetGenerator(
+                user_kpoints_settings={"reciprocal_density": 100},
+                user_incar_settings={
+                    "EDIFF": 1e-7,
+                    "LAECHG": False,
+                    "LVTOT": False,
+                    "LREAL": False,
+                    "ALGO": "Normal",
+                    "LWAVE": False,
+                },
+            )
+        )
+    )
+    bs_maker: BaseVaspMaker = field(
+        default_factory=lambda: NonSCFMaker(
+            input_set_generator=NonSCFSetGenerator(
+                user_kpoints_settings={"reciprocal_density": 400},
+                user_incar_settings={
+                    "LWAVE": True,
+                    "ISYM": 0,
+                },
+            )
+        )
+    )
+
+    def make(self, structure: Structure, prev_vasp_dir: str | Path | None = None):
```

Comment:
Do you have a suggestion with regard to code reuse? I think I simply copied this part and just adapted the part above.

Reply:
You could just subclass the `UniformBandStructureMaker` and remove the make function entirely?

Reply:
Jimmy has done it this way in the phonon workflow and means we don't get too many makers. The line between adding a new maker vs reusing the old ones is quite ambitious though, so happy either way.

Reply:
I like the constant in this case. I was thinking about subclassing but this would have added another layer of complexity as well.

Reply:
(btw, I also like that we still get the VASP dos in this case. It also allows to compare Lobster and VASP dos if necessary. thus, I will keep the higher NEDOS setting)